src/zap/zapimage.cpp

```diff
@@ -1025,8 +1025,8 @@ HANDLE ZapImage::GenerateFile(LPCWSTR wszOutputFileName, CORCOMPILE_NGEN_SIGNATU
     {    
         // Write the debug directory entry for the NGEN PDB
         RSDS rsds = {0};
-        
-        rsds.magic = 'SDSR';
+
+        rsds.magic = (DWORD)('S' * 0x1000000 +  'D'*0x10000 + 'S' * 0x100 + 'R');
```

Comment:
Nit: Inconsistent spacing around `*`

Reply:
This is what the same magic looks in other places: https://github.com/dotnet/coreclr/search?q=0x53445352&unscoped_q=0x53445352 . Maybe nice to use the same pattern.

Reply:
sure